code/game/objects/structures/lavaland/geyser.dm

```diff
@@ -56,6 +56,8 @@
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "plunger"
 
+	slot_flags = ITEM_SLOT_MASK
```

Comment:
Please give it welder vision. You are putting it over your face

Reply:
light is absorbed through the handle and deposited into your eyes through the advanced photon conducting red suction cup